.github/dependabot.yml

```diff
@@ -0,0 +1,13 @@
+version: 2
+updates:
+- package-ecosystem: npm
+  directory: "/"
+  schedule:
+    interval: weekly
+    time: "04:00"
+  open-pull-requests-limit: 10
+  labels:
+  - skip-changelog
+  - dependencies
+  versioning-strategy: increase
+  rebase-strategy: disabled
```

Comment:
```suggestion
  rebase-strategy: disabled
   ignore:
       - dependency-name: "eslint*"
       - dependency-name: "babel-eslint"
```

Reply:
👍

Reply:
Let's add this change and merge as you wish